src/App.js

```diff
@@ -5,6 +5,8 @@ import CreateTrip from './components/pages/CreateTrip'
 import TripList from './components/pages/TripList'
 import TripsData from './data/tripsdata.json'
 import Icons from './common/Icons'
```

Comment:
Did you delete the Icon for the code review to work? Otherwise Icons is declared but never read